### PR TITLE
Removes Code from Feature-Toggles

### PR DESCRIPTION
**Pull request by jwicks31**
Removed Functions, README.md  and TS typings
- [x] createExpressMiddleware
- [x] getActiveFeatures
- [x] getQueryFeatures
- [x] mergeFeatures
- [x] removeFeatures
- [x] isActive
- [x] getReqQueryFeatures
- [x] getBrowserQueryFeatures
- [x] getCurrentActiveFeatures
- [x] getIsEnabled
- [x] updateFeaturesWithQuery

Removed devDependencies:
* [x] supertest
* [x] express

Removed Dependencies
* [x] url-parse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ If the feature is enabled then the _activeComponent_ will render else it renders
 
 Feature takes these **props**
 
-* name = ""
-* inactiveComponent = noop
```
Comment (kennylavender) [inline README.md]: I've noticed that everytime you change something in the readme, your prettier version or editor changes all of these bullet points from `*` to `-`. It shouldn't be doing that, it's creating a bunch of unnecessary changes. Maybe deleting your node modules folder and re-running npm install will fix this if its prettier version related. If it's your editor maybe you can debug it.

**Comment by kennylavender** [comment]
I believe the entire src/test/fixtures folder can be deleted, if you can delete those and make sure the tests still pass.

File: README.md [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 ## Version 2
-This is version 2, it contains many breaking changes from version 1. 
+
+This is version 2, it contains many breaking changes from version 1.
 
 ## Requirements
 
 React Feature Toggles attempts to satisfy the following requirements:
 
-* Universal - server and client side
```
Comment (kennylavender) [inline README.md]: This line shouldn't be removed. Just curious as to why you thought it should be removed?

File: README.md [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 * Conditionally execute code based on the presence or absence of a specific feature
 * Toggle features on with url parameters
 * Feature Dependency - if a feature depends on a another feature that is disabled, then neither of them should execute
```
Comment (kennylavender) [inline README.md]: this line can be removed, this is handled by feature-toggles, not this package.

File: README.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
+
+This is version 2, it contains many breaking changes from version 1.
 
 ## Requirements
 
 React Feature Toggles attempts to satisfy the following requirements:
 
-* Universal - server and client side
 * Conditionally execute code based on the presence or absence of a specific feature
 * Toggle features on with url parameters
```
Comment (kennylavender) [inline README.md]: this line can be removed, its handle by feature toggles, not this repo

File: README.md (hunk 2 above)
Comment (jwicks31) [reply README.md]: I didn't really think enough about React being fine to render on the server side. I did it because I thought that was one of the issues we were having with the combined library, but I didn't really think enough about it. My bad.

Comment (kennylavender) [reply README.md]: Cool, no worries :) Just wanted to make sure I didn't give you the impression it was no longer universal

**Comment by kennylavender** [comment]
Looks good! Thanks!
